docs/preview/02-Features/powershell/azure-sql.md

```diff
@@ -55,27 +55,18 @@ PS> RunDatabaseScript -ServerName "my-server-name" -DatabaseName "my-database-na
 
 ### Adding SQL scripts so they can be picked up by the script
 
-1. In the location where you want to run the script add the folder "sqlScripts".
-
-2. Within this folder there should be by default the `CreateDatabaseVersionTable.sql`-file, containing the script to create the initial version table:
-
-```sql
-CREATE TABLE [dbo].[DatabaseVersion]
-(
-    [CurrentVersionNumber] INT NOT NULL,
-    [MigrationDescription] [nvarchar](256) NOT NULL,
-    CONSTRAINT [PKDatabaseVersion] PRIMARY KEY CLUSTERED
-    ( 	
-        [CurrentVersionNumber] ASC
-    )
-    WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON)
-)
-```
+1. In the location where you want to run the script add the folder `sqlScripts`.
```

Comment:
This is actually configurable, maybe that should be mentioned here. ("As an example, we use ..."). or something like that.

Reply:
Made some changes to make this more clear